src/logic/nfts/collections.ts

```diff
@@ -192,7 +193,11 @@ export function fromCollectionsFragment(
       contractAddress: fragment.contractAddress,
       category,
       activeOrderId:
-        fragment.activeOrder && !isOrderExpired(fragment.activeOrder.expiresAt)
+        fragment.activeOrder &&
+        ((caller &&
+          caller === fragment.owner.address.toLowerCase() &&
+          fragment.activeOrder.expiresAt.length === 13) ||
+          !isOrderExpired(fragment.activeOrder.expiresAt))
```

Comment:
WDYT about moving this logic into a new function? If we need to change something else, we should change it in four different places.